ui_develop

yuikateraoka opened the pull request:
I delated 1st ui folders, and also created new ui  folders.

yuikateraoka committed:
test

yuikateraoka committed:
delated ui files

yuikateraoka committed:
add ui file

yuikateraoka committed:
add package-lock.json

yuikateraoka committed:
add yarn.lock

yuikateraoka committed:
Modified TwitterTimeline,teens

yuikateraoka committed:
Modified TwitterTimeline,youth

yuikateraoka committed:
modified

seiya303 committed:
urlに#が表示されるのを消しました

yuikateraoka committed:
delated all components

yuikateraoka committed:
added nem components

seiya303 commented on ui/src/router.js:
```diff
@@ -6,6 +6,7 @@ import Teens from './components/Teens.vue'
 Vue.use(Router)
 
 export default new Router({
+  mode: 'history',
```
historyモードを追加してurlに＃が表示されるのを無くしました。

seiya303 committed:
サーバー構造をリファクタリングしました。 (#5)

* refactored ui directory to implement a server rendering

* modified router.js to redirect to home directory

* added publicpath so that index.html can import files from subdirectory

* added a base /teens/ so that the router can recognize the subdirectory

[in summary]
this commit refactors the project by which divides ui directory to two directory, youth and teens directory.

[purpose]
So that the project matches to the format of xserver for deployment.

yuikateraoka committed:
test

yuikateraoka committed:
ファイル修正

yuikateraoka committed:
file delete

yuikateraoka committed:
add NavigationMenu

yuikateraoka committed:
add

yuikateraoka committed:
git add

yuikateraoka committed:
git add

yuikateraoka committed:
git add .

yuikateraoka committed:
git add .

yuikateraoka committed:
git add

yuikateraoka committed:
git add

yuikateraoka committed:
git add

yuikateraoka committed:
git add

yuikateraoka committed:
git add

yuikateraoka committed:
git add .

yuikateraoka committed:
git add .

yuikateraoka committed:
git add .

seiya303 committed:
changed the url style of link to youth in teens page

seiya303 commented on teens_ui/src/components/teens/NavigationMenu.vue:
```diff
@@ -73,7 +73,7 @@
             text 
             class="hidden-xs-only"
-            :href="'/https://sinergy2021.org'"
+            :href="'https://sinergy2021.org'"
```
[変更点]頭にあるバックスラッシュを消しました。

v-btnにある:hrefに入るurlとして、頭にバックスラッシュを入れるとBASE_URL下のサブディレクトリとしてみなされるので、:href内にBASE_URL（今回はhttps://sinergy2021.org ）を入れる場合は、頭にバックスラッシュを入れない形でお願いします。

seiya303 commented:
追記->[こちらのPR](https://github.com/YSP-SINERGY/xserver-sinergy2021-event-website/pull/1#issue-1070230800)

[主な変更点]
- ディレクトリの構造を切り分けました。
uiディレクトリをteens_uiとyouth_uiに分割
  - 変更理由: 
  XServerにデプロイ時にサブディレクトリが一つのindex.htmlを基点としてフォルダ単位で認識されるため、youth用のページ、teens用のページにそれぞれindex.htmlをビルドして作成するため、便宜上二つのフォルダに切り分けました。
  - 関連イシュー：
    - https://github.com/YSP-SINERGY/xserver-sinergy2021-event-website/issues/3

- デザインを改善しました。
ミーティング時に指摘を受けたUIをUXの観点で改善
  - 変更理由：
  シナジーのコンセプトに合ったデザイン、また、youthとteensそれぞれの年代層にあったデザインを実現するため。また、細かい所でUX上気になる部分をなくすため。
  - 関連イシュー：
    - https://github.com/YSP-SINERGY/xserver-sinergy2021-event-website/issues/4

- ページの読み込み速度を改善しました。
ページ内でも特に画像の読み込み速度が遅かったので、その点を改善
  - 変更理由：
  読み込みの遅さが著しく目立っていて、その点をストレスに感じさせない程度に改善
  - 関連イシュー：
    - https://github.com/YSP-SINERGY/xserver-sinergy2021-event-website/issues/2

seiya303 commented:
- https://github.com/YSP-SINERGY/xserver-sinergy2021-event-website/issues/4
- https://github.com/YSP-SINERGY/xserver-sinergy2021-event-website/issues/2
こちらの２点に関しましては、今後改善の余地がまだあるのでひとまずクローズせずに残しておきたいと思います。

seiya303 committed:
changed the tab title to sinergy

seiya303 commented on youth_ui/vue.config.js:
```diff
@@ -2,4 +2,10 @@ module.exports = {
+    index: {
+      entry: "src/main.js",
+      title: "sinergy"
+    }
```
タブタイトルをsinergyに変えました。ドメイン名がsinergyとなっていて、iで統一するのが良いと判断したためです。

seiya303 reviewed (approved):
- リンクに異常がないか
- デザイン面で気になる点がないか
- 非機能面で著しく目立つ点はないか
を主に確認しましたが、大きな問題点はなさそうなので承認します。
